### ISSUE TITLE
Improve guide to writing blog articles

### ISSUE THREAD

**Issue by sftim** [kubernetes/website#44123]
**This is a Feature Request**

**What would you like to be added**
- Based on feedback from people who have written blog articles, improve our [guide](https://kubernetes.io/docs/contribute/new-content/blogs-case-studies/).

To do this, you would:
- find some people who have written blog articles recently
  - try to pick a mixture of people writing their first article and people who have already contributed to the blog
  - also have a mix of release-related announcements, other project announcements, and articles that are neither
- talk to those people to find out what they found hardest about the process
- based on that feedback, propose one or more pull requests to make the documentation better

**Why is this needed**
- people writing for our blog find the process hard
- we also have very little blog editorial capacity - see https://github.com/kubernetes/community/blob/master/sig-docs/blog-subproject/README.md (as of 2023-11-28)
  - so it's easier if the articles that come in need less review, and there are fewer round-trips between authors and reviewers

**Comments**
/sig contributor-experience
/language en

**Comment by dipesh-rawat** [comment]
I've noticed a couple of additions that could be made to our blog guide based on my reviews of the blog PRs. Contributors could benefit from clearer instructions on incorporating images in blog using Hugo shortcode, addressing issues such as image format and size. Additionally, it would be helpful to include guidance on mirroring blogs between [k8s.dev/blog](https://www.kubernetes.dev/blog/) and [k8s.io/blog](https://www.kubernetes.io/blog/) for a smoother overall experience.

**Comment by dipesh-rawat** [comment]
This would be a valuable enhancement. Improving our guide based on feedback will make the blogging process smoother for contributors and aligns with our goal to streamline reviews. Thanks for raising this @sftim!

/triage accepted
/priority important-longterm
_(Please feel free to adjust the priority as needed, if the SIG consensus leans toward a different priority)_

**Comment by kumarankit999** [comment]
Yes @sftim, it would be a great idea. People can write the blog to enhance the kubernetes documentation.

**Comment by Rajdeep1311** [comment]
Can I submit my blogs? I have recently written 2 blogs, one on my experience in the first ever KubeCon attended and the other on my experience as CNCF zero to merge incubator?

**Comment by Gauravpadam** [comment]
Sure @Rajdeep1311, We always appreciate blog contributions

**Comment by Rajdeep1311** [comment]
Please, you can work @haripriya9647

**Comment by Rajdeep1311** [comment]
@haripriya9647 , If you want ma'am we both can work on this

**Comment by haripriya9647** [comment]
yes sure

**Comment by haripriya9647** [comment]
Hello @sftim 
I am planning to work collaboratively with @Rajdeep1311 on this issue. The two of us have decided to create a form that includes all the questions about the blog guides, and based on the response we will be able to identify the pain points and analyze appropriate solutions.
As soon as the form has been created, we will head to the sig-docs-blog slack channel to gather a list of recent contributors from different domains, and we will contact them and ask them to fill out the form, as well as posting the form in blog-related channels.

**Comment by Rajdeep1311** [comment]
Any replies @sftim sir?

**Comment by sftim** [comment]
> I am planning to work collaboratively with @Rajdeep1311 on this issue. The two of us have decided to create a form that includes all the questions about the blog guides, and based on the response we will be able to identify the pain points and analyze appropriate solutions.
> As soon as the form has been created, we will head to the sig-docs-blog slack channel to gather a list of recent contributors from different domains, and we will contact them and ask them to fill out the form, as well as posting the form in blog-related channels.

That sounds fine.

I think @david-martin has also made a start here.

**Comment by david-martin** [comment]
Some updates on this.

> using Hugo shortcode

https://github.com/kubernetes/website/pull/44967

> clearer instructions on incorporating images in blog
> addressing issues such as image format and size.

There is a diagram guide already at https://kubernetes.io/docs/contribute/style/diagram-guide/

@dipesh-rawat Is there something specific missing from that, or is the discoverability of that page a problem when writing blogs?

I did get some specific suggestions from @sftim, which we can add to that diagram guide if it makes sense and isn't sufficiently covered already:

> - PNG with alpha for anything that has a shape, like a logo, and doesn't have a vector version
> - JPEG for photos, either original size or scaled down to 1920px maximum dimension
> We can use optimization tools suitable to the format if images seem particularly large.


Also, regarding:

> mirroring blogs between [k8s.dev/blog](https://www.kubernetes.dev/blog/) and [k8s.io/blog](https://www.kubernetes.io/blog/)

I'm not familiar with the process here, but a suggestion was to write a blog article on how to do this and publish it.
@dipesh-rawat Is there someone you'd recommend reaching out to that has this knowledge, and when to use 1 vs. the other, and when you'd consider mirroring?

**Comment by david-martin** [comment]
> > mirroring blogs between [k8s.dev/blog](https://www.kubernetes.dev/blog/) and [k8s.io/blog](https://www.kubernetes.io/blog/)

@Gauravpadam Perhaps this is something you have some notes about?
I see you added https://github.com/kubernetes/website/pull/44722 to mirror https://github.com/kubernetes/contributor-site/pull/424

**Comment by Gauravpadam** [comment]
Sure @david-martin, I'll be glad to help with the process. I'll reach out to you on slack.

**Comment by a-mccarthy** [comment]
Hello, reaching out to see what the current state of this issue is? I was thinking of ways we could ask for feedback from release feature blog authors (current cycle 1.31), so maybe we can combine efforts?

**Comment by david-martin** [comment]
@a-mccarthy I know there were a few small updates to the docs around contributing. In that sense, improvements were made.

> find some people who have written blog articles recently ... talk to those people ... based on that feedback, propose one or more pull requests

However, I'm not aware of what has progressed in this area.
This [comment](https://github.com/kubernetes/website/issues/44123#issuecomment-1866171262) from @haripriya9647 (with @Rajdeep1311 mentioned) suggested a form was being created to gather feedback from blog authors. I would defer to either of them on how that has progressed.
